docs/ispc.rst

```diff
@@ -4252,6 +4252,18 @@ use Newton-Raphson.
     float rcp_fast(float v)
     uniform float rcp_fast(uniform float v)
 
+The ``fmodf()`` functions compute the floating-point remainder of the division
```

Comment:
`f` suffix is C language mangling of `float` argument. I.e. see: https://man7.org/linux/man-pages/man3/fmod.3.html
 
So C has `fmod(double, double)`, `fmodf(float, float)`, and `fmodl(long double, long double)`.

C++ has overloaded `fmod()` function: https://en.cppreference.com/w/cpp/numeric/math/fmod

As long as we have overloading in the language, we should name it `fmod()`, not `fmodf()`.

Reply:
Fixed